pom.xml

```diff
@@ -5,17 +5,18 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.580.1</version>
+    <version>2.11</version>
   </parent>
 
   <artifactId>openid</artifactId>
   <version>2.2-SNAPSHOT</version>
   <packaging>hpi</packaging>
 
   <properties>
-    <findbugs-maven-plugin.version>3.0.1</findbugs-maven-plugin.version>
     <!--TODO: Enforce when the code becomes 100% clean-->
     <findbugs.failOnError>false</findbugs.failOnError>
+    <jenkins.version>1.586</jenkins.version>
```

Comment:
🐜 for the non-LTS version. If you need an update due to whatever reason use 1.596 or 1.609